vizro-core/src/vizro/static/css/bootstrap_overwrites.css

```diff
@@ -29,6 +29,11 @@ All the HEX values starting with --text-code are taken from the Github code high
 }
 
 /* CARDS */
+.card {
+  height: 100%;
+  width: 100%;
+}
```

Comment:
This was previously defined inside our vizro-bootstrap theme, but I took it out now and decided that it should live inside vizro as an overwrite. Bootstrap cards resize according to their content, so I think we shouldn't change that default behaviour. However, inside vizro, we can set that as a default.

Reply:
Is there a straightforward way to make the `kpi_card` content dynamically collapse/expand when the app is zoomed in or out? @maxschulz-COL pointed out this issue a few days ago. Specifically, when the app is zoomed out (e.g. to 80%), most of the app's content scales accordingly, but the `kpi_card` content does not adjust as expected.

I tested this with the first kpi_card example from our [documentation](https://vizro.readthedocs.io/en/stable/pages/user-guides/figure/), and the issue persists on this branch as well.

This is not an urgent issue, but if it's an easy enhancement to implement, we should consider addressing it.

Reply:
In general, we don't optimise for zoom in/zoom out views. Our apps should work well for 100% full page views (which can definitely be improved). There are several reasons for this, but the most important one is font hierarchy 😬 D. has set fixed sizes for fonts and spacings and all our components are scaled according to these. As soon as you zoom in/out, you change the ratio of these, so we don't guarantee that it looks good on all scaled views. 

However, the KPI Card is a separate issue for me. We can definitely tackle that, but I wouldn't do it in this PR as it's not really related. @maxschulz-COL - could you create a separate issue for this and explain what your expected behaviour is? In general the reason why the KPI card doesn't react to your zoom in / zoom out is this line   `font-size: 3.6vh;`. The font-size is dynamic depending on the spacing it gets. You can also hardcode the value e.g. say `font-size: 16px` and then it would change based on the zoom in / zoom out, but currently I am not sure what would look well on **all bootstrap themes**. We would have to try things out 👍

Reply:
Top, I'll write the issue